ang/civiawards-payments-tab/directives/payments-table.directive.html

```diff
@@ -0,0 +1,30 @@
+<div class="civiawards__reviews-tab__table-container panel panel-default">
```

Comment:
If we are using `civiawards__reviews-tab__table-container`, we should give it a more generic name.